angular.json

```diff
@@ -0,0 +1,124 @@
+{
```

Comment:
I haven't had the chance to use Angular, so I will need more time than the allotted code review time to give a constructive review! I'm sure I'll be learning a lot from this code. :)